.github/ISSUE_TEMPLATE/questions_and_others.md

```diff
@@ -0,0 +1,10 @@
+---
+name: "❓ Questions and Others"
+about: Questions regarding Klaytn Contracts or other issues not related to bug nor feature request
```

Comment:
regarding Klaytn Dex contract would be nice

Reply:
Done